ecosystem/sep-0006.md

```diff
@@ -1462,6 +1463,22 @@ Each object in the `transactions` array should have the following fields:
 | `amount`  | string | The amount sent back to the user for the payment identified by `id`, in units of `amount_in_asset`.                                                                                                                                                                  |
 | `fee`     | string | The amount charged as a fee for processing the refund, in units of `amount_in_asset`.                                                                                                                                                                                |
 
+### Fee Details Object Schema
+
+| Name        | Type   | Description                                                                                                                                                           |
+| ----------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `total`     | string | The total amount of fee applied.                                                                                                                                      |
+| `asset`     | string | The asset in which the fee is applied, represented through the [Asset Identification Format](sep-0038.md#asset-identification-format).                                |
+| `breakdown` | array  | (optional) An array of objects detailing the fees that were used to calculate the conversion price. This can be used to datail the price components for the end-user. |
```

Comment:
I know its redundant to have `fee_details.details` but I think its better than `breakdown`. Can we update the naming in this structure to match SEP-38's fee object as closely as possible?

Reply:
That's a good point, I was going to reuse exact same object, but forgot to change the field name back